components/portal/widgets/partials/type__action-items.html

```diff
@@ -24,11 +24,17 @@
     <!-- Loading case -->
     <loading-gif data-object="actionItems" ng-show="loading"></loading-gif>
 
-    <!-- Error case -->
+    <!-- Error case (endpoint failure) -->
     <div class="error" layout="row" layout-align="start center" ng-show="error">
       <div><md-icon class="md-warn">warning</md-icon></div>
       <span>There was an issue loading the information. Click See all for more info.</span>
     </div>
+
+    <!-- Error case (non-number quantity for all items) -->
+    <div class="error" layout="row" layout-align="start center" ng-show="errorQuantity">
+      <div><md-icon class="md-warn">warning</md-icon></div>
+      <span>One or more items failed to retrieve quantity data. Click see all for more info.</span>
```

Comment:
Will the widget launch bar label reliably be labeled "See all"?

Reply:
No, good catch. The launch text is configurable (as with all widgets) -- will update.